src/testRunner/unittests/incrementalParser.ts

```diff
@@ -160,7 +160,7 @@ describe("unittests:: Incremental Parser", () => {
         const oldText = ts.ScriptSnapshot.fromString(source);
         const newTextAndChange = withInsert(oldText, semicolonIndex, "/");
 
-        compareTrees(oldText, newTextAndChange.text, newTextAndChange.textChangeRange, 4);
+        compareTrees(oldText, newTextAndChange.text, newTextAndChange.textChangeRange, 0);

```

Comment:
what is this 4th parameter?

Reply:
0 means "all AST nodes were reused", IIRC; so it being a parse error before meant we weren't reusing the AST in incremental parsing.

Reply:
The number of elements that could be reused. This was `0` prior to the original PR and only changed due to the additional diagnostics that were added during parse. Now that these diagnostics are produced during check, this needs to be reverted to the prior value.